tests/Kiota.Tests/KiotaHostTests.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using Xunit;
+using System.Threading.Tasks;
+using System.CommandLine;
+
+namespace Kiota.Tests
+{
+    public class KiotaHostTests
+    {
+        [Fact]
+        public async Task ThrowsOnInvalidOutputPath() {
+            await KiotaHost.GetRootCommand().InvokeAsync(new string[] { "-o", "A:\\doesnotexist" });
+        }
+        [Fact]
+        public async Task ThrowsOnInvalidInputPath() {
+            await KiotaHost.GetRootCommand().InvokeAsync(new string[] { "-d", "A:\\doesnotexist" });
+        }
+        [Fact]
+        public async Task ThrowsOnInvalidInputUrl() {
+            await KiotaHost.GetRootCommand().InvokeAsync(new string[] { "-d", "https://nonexistentdomain56a535ba-bda6-405e-b5e2-ef5f11bf1003.net/doesnotexist" });
+        }
+        [Fact]
+        public async Task ThrowsOnInvalidLanguage() {
+            await KiotaHost.GetRootCommand().InvokeAsync(new string[] { "-l", "Pascal" });
+        }
+        [Fact]
+        public async Task ThrowsOnInvalidLogLevel() {
+            await KiotaHost.GetRootCommand().InvokeAsync(new string[] { "-ll", "Dangerous" });
```

Comment:
This option is defined with two hyphens (`--ll`). Which one is the intended alias?

My understanding was that two hyphens were for long descriptive names and one hyphen was for a single letter short options. There is also a convention that single hyphen options can be combined in UNIX systems (e.g. `-rf` instead of `-r -f`). That said, I am not sure if dotnet defines its own convention.

Reply:
nice catch! I was told anything longer than a single letter should be using two hyphens. But those are old remnants of my system and OS classes and I might be wrong. Regardless, this parsing library (system.commandline) seems to be fairly tolerant with hyphens. Added the missing hyphen just in case.